src/Schema.php

```diff
@@ -692,10 +692,10 @@ protected function findColumns(TableSchemaInterface $table): bool
             (SELECT nspname FROM pg_namespace WHERE oid = COALESCE(td.typnamespace, tb.typnamespace, t.typnamespace)) AS type_scheme,
             a.attlen AS character_maximum_length,
             pg_catalog.col_description(c.oid, a.attnum) AS column_comment,
-            a.atttypmod AS modifier,
-            a.attnotnull = false AS is_nullable,
-            CAST(pg_get_expr(ad.adbin, ad.adrelid) AS varchar) AS column_default,
-            coalesce(pg_get_expr(ad.adbin, ad.adrelid) ~ 'nextval',false) $orIdentity AS is_autoinc,
+            information_schema._pg_truetypmod(a, t) AS modifier,
+            NOT (a.attnotnull OR t.typnotnull) AS is_nullable,
+            COALESCE(t.typdefault, pg_get_expr(ad.adbin, ad.adrelid)) AS column_default,
+            COALESCE(pg_get_expr(ad.adbin, ad.adrelid) ~ 'nextval', false) $orIdentity AS is_autoinc,
```

Comment:
Good solution. It's adopt from SQL to internal functions?
As https://gemfury.com/schabrolles/rpm:postgresql96-server/-/content/usr/pgsql-9.6/share/information_schema.sql

Reply:
Yes, the functions `_pg_numeric_precision` and `_pg_numeric_scale` have the same code and it looks more accurate to use them.
https://github.com/postgres/postgres/blob/master/src/backend/catalog/information_schema.sql